releases-index.json

```
@@ -8,7 +8,7 @@
             "latest-runtime": "7.0.0-rc.2",
             "latest-sdk": "7.0.100-rc.2.22477.23",
             "product": ".NET",
-            "support-phase": "rc",
+            "support-phase": "go-live",
```

don't introduce the breaking changes now

```suggestion
            "support-phase": "rc",
```